### PR TITLE
fill the section on who the course is for

### PR DESCRIPTION
**Pull request by wikfeldt**
to review:
- any other prerequisites?
- should the text be more "welcoming" somehow by not only pointing out what is needed but also what will be learned?

### CODE REVIEW

**Comment by wikfeldt** [comment]
forgot, this PR hopefully fixes #3

File: content/index.rst [hunk 1]
```diff
@@ -54,9 +54,22 @@ TODO Intro
...
+participants have some prior experience with or knowledge of the
+following topics:
+
+ - General concepts: distributed memory parallelism, MPI process model
+ - Communicators
+ - Point-to-point communication
+ - Non-blocking communication
+ - Collective communication
+ - MPI datatypes
+
```
Comment (robertodr) [inline content/index.rst:71]: Should we point to the material for the PDC introductory course?

**Review by robertodr — APPROVED** [review]
Thanks Thor! We can list what will be learned under the "About" section, I think.

File: content/index.rst (hunk 1 above)
Comment (wikfeldt) [reply content/index.rst:71]: good idea, i've added links to the PDC course and the upcoming SNIC course

**Comment by wikfeldt** [comment]
no further review needed i think, self-merging

**Pull request merged**